src/components/Header/styles.ts

```diff
@@ -0,0 +1,21 @@
+import Styled from 'styled-components/native';
+
+export const HeaderConteiner = Styled.View`
+    width: 95%;
+    flex-direction: row;
+    padding-horizontal:18px;
+    padding-vertical:42px;
+    align-items: center;
+`;
+export const TittleText = Styled.Text`
+    color: white;
```

Comment:
declare a theme file: 

src/utils/theme.ts

export const theme = {
   colors: {
      white: "#ffffff",
      black: ....
      }
}

import and use it